kubernetes/assets/dashboards/kubernetes_pods.json

```diff
@@ -75,22 +75,22 @@
                 "type": "timeseries",
                 "requests": [
                     {
-                        "q": "sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:failed,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:pending,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:running,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:succeeded,$scope,$statefulset,$daemonset,$job}",
+                        "q": "sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:failed,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:pending,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:running,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:succeeded,$scope,$kube_stateful_set,$kube_daemon_set,$job}",
                         "metadata": [
                             {
-                                "expression": "sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:failed,$scope}",
+                                "expression": "sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:failed,$scope}",
```

Comment:
same here since it's the same metric. You can check the tags available on a metric here: https://app.datadoghq.com/metric/summary?filter=kubernetes_state.pod.status_phase&metric=kubernetes_state.pod.status_phase - search the tag menu with `kube_` and you'll see what i'm talking about.